internal/mode/static/telemetry/collector.go

```diff
@@ -160,6 +179,9 @@ func (c DataCollectorImpl) Collect(ctx context.Context) (Data, error) {
 		FlagNames:         c.cfg.Flags.Names,
 		FlagValues:        c.cfg.Flags.Values,
 		NGFReplicaCount:   int64(replicaCount),
+		// maybe SnippetValues?
```

Comment:
Thoughts on a different name?

Reply:
Maybe just `SnippetsFiltersDirectives` and `SnippetsFiltersDirectivesCount`.

I know we are also considering the context, but really we're just counting the directives, and the context is metadata. Easier to read and understand too

Reply:
Sure that makes sense to me